goal_x: 0.0
goal_y: 6.0
goal_z: 0.0
